build.sbt

```diff
@@ -122,18 +122,31 @@ lazy val root = Project(id="fgbio", base=file("."))
     libraryDependencies ++= Seq(
       "org.scala-lang"            %  "scala-reflect"  % scalaVersion.value,
       "org.scala-lang"            %  "scala-compiler" % scalaVersion.value,
-      "org.scala-lang.modules"    %% "scala-xml"      % "1.0.6",
-      "com.fulcrumgenomics"       %% "commons"        % "0.7.0",
-      "com.fulcrumgenomics"       %% "sopt"           % "0.7.0",
+      "org.scala-lang.modules"    %% "scala-xml"      % "1.2.0",
+      "org.scala-lang.modules"    %% "scala-collection-compat" % "2.1.1",
+      "com.fulcrumgenomics"       %% "commons"        % "0.8.0-c052fc9-SNAPSHOT",
```

Comment:
confirming that we'll use https://github.com/fulcrumgenomics/commons/pull/54 when it is merged?  In particular `View`.